### PR TITLE
STIX Shifter that works with CP4S 1.3.0.1 as Proxy Host with QRadar

### PR DESCRIPTION
**Pull request by barvhaim**
Modifications to STIX Shifter to make it work as proxy host mode with QRadar and CP4S 1.3.0.1

### CODE REVIEW

**Comment by barvhaim** [comment]
at file `stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix.py` line 32 `return json.dumps(results, indent=4, sort_keys=False)` - without this change, the result will not appear in CP4S 1.3.0.1, however, this change cause the QRadar test not to pass. @yurii-klymenko should the test be changed? what's your recommendation?

File: stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix.py [hunk 1]
```diff
@@ -29,4 +29,4 @@ def translate_results(self, data_source, data):
         except Exception as ex:
             raise TranslationResultException("Error when converting results to STIX: {}".format(ex))
         
-        return results
+        return json.dumps(results, indent=4, sort_keys=False)
```
Comment (mdazam1942) [inline stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix.py:32]: we can't change this back to json strings as that might effect how UDI pipeline handles results. you may need to handle this into the proxy module level.

Comment (barvhaim) [reply stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix.py:32]: okay we'll take a look

**Comment by mdazam1942** [comment]
@barvhaim I'm not sure what's the implication on the UDI side of it if we don't handle unexpected_params the way it is now. @yurii-klymenko would need to investigate more into it. since he is in vacation for a week. Is there any way you could handle those unexpected parameters in proxyhost class or even in proxy module? Just wanted to mention, you can make as many changes you want in proxy module or proxyHose if needed.

**Comment by barvhaim** [comment]
since no proxy mode on CP4S 1.4 I think it's not relevant anymore, we'll modify the official connectors

**Pull request closed without merging**